templates/php-resque-staggered-restart.sh.erb

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -eo pipefail
+
+# Blocks until started.
+systemctl restart php-resque-flush
+
+for service in $(systemctl --all | grep 'php-resque@' | awk '{print $1}')
```

Comment:
Would it be simpler to remove the loop and do `systemctl restart "php-resque@*"`?

Reply:
Lol yes, could do that :D

Reply:
The staggered restart script got so short that I removed it, moved into the cronjob.